beacon-chain/blockchain/forkchoice/process_attestation.go

```diff
@@ -99,8 +98,8 @@ func (s *Store) OnAttestation(ctx context.Context, a *ethpb.Attestation) error {
 	}
 
 	// Verify Attestations cannot be from future epochs.
-	if err := helpers.VerifySlotTime(baseState.GenesisTime, tgtSlot); err != nil {
```

Comment:
Can we keep using helpers.VerifySlotTime?

Reply:
https://github.com/ethereum/eth2.0-specs/issues/1502 calls for explicit use of `slot` instead of `time`, such fix brings complexity due to our familiarity using time shift tolerance. It's not clear ot me why we have to use `slot` instead of `time` to verify future objections. I'm tempted to keep using `time` until we understand the why